### PR TITLE
Fix KeyManager::getDefaultKeyContent() method

### PR DESCRIPTION
**Pull request by mradcliffe**
Per Pull Request #10.

Is there a different method that should be called, and what should the expected output be?

### CODE REVIEW

**Comment by mradcliffe** [comment]
I should have done this with #15 as well. Iposted JSON to github's api to convert it from issue to pull request so that you don't have to close pull request and issue.

**Comment by mradcliffe** [comment]
I looked at the docs for getKeyValue() and it looks like this is what getDefaultKeyContent() does as well so I modified the method to do that.

File: tests/src/KeyManagerTest.php [hunk 1]
```diff
@@ -103,14 +103,31 @@ public function testGetDefaultKey() {
    * Test load of defaul key content.
    *
    * @group key
-   *
-   * @todo There is no Key::getContents() method so this will return an error.
    */
   public function testGetDefaultKeyContent() {
+    // On the first run, config storage will return NULL.
```
Comment (mradcliffe) [inline tests/src/KeyManagerTest.php]: Much of this is borrowed from `testGetDefaultKey`. Would it be better to consolidate the test data into a provider method?

**Comment by nerdstein** [comment]
Reviewed and merged

**Pull request merged**